Microsoft.HealthVault/Connection/SessionCredential.cs

```diff
@@ -20,5 +22,13 @@ public class SessionCredential
         /// Gets or sets the shared secret.
         /// </summary>
         public string SharedSecret { get; set; }
+
+        /// <summary>
+        /// Gets or sets the created time of the token.
+        /// </summary>
+        /// <value>
+        /// The created.
+        /// </value>
+        public DateTime ExpirationUtc { get; set; }
```

Comment:
DateTime is just a terrible class that should only be used as a last resort. Like if you hate NodaTime and really need to represent a local time. DateTimeOffset is much more suited to represent an "instant" in time like what you're using it for here. The reason being that it has the offset baked into the value, which in this case will be 0. DateTime has a bunch of nightmarish bugs around comparison, conversion and serialization. I can talk more in person about it if you like.

---
In reply to: [115805292](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115805292) [](ancestors = 115805292,115796836)